README.md

```diff
@@ -90,83 +91,149 @@ const requestResults = await interBTC.redeem.request(amount, btcAddress);
 
 At this point, one more more Vaults will send BTC to the address specified within 24 hours.
 
+### Creating an AccountId Instance
+
+Certain API calls require a parameter of type `AccountId`, which represents the Polkadot/Kusama account and can be instantiated as follows
+
+```ts
+import { AccountId } from "@polkadot/types/interfaces/runtime";
+
+const activeStakedRelayerId = await interBTC.api.createType(
+    "AccountId",
+    "5Ck5SLSHYac6WFt5UZRSsdJjwmpSZq85fd5TRNAdZQVzEAPT"
+);
+const feesEarnedByActiveStakedRelayer = await interBTC.stakedRelayer.getFeesEarned(
+    activeStakedRelayerId
+);
+```
+
 ### More Examples
 
-There are plenty more examples how to use this library. Best is to take a look at the integration tests: https://github.com/interlay/interbtc-api/tree/master/test/integration
+There are many examples in the integration tests, showing how to use this library. Take a look at them here: https://github.com/interlay/interbtc-api/tree/master/test/integration
 
 ## API Documentation
 
 Please check the documentation at https://docs.interlay.io/interbtc-api/#/classes/DefaultInterBtcApi
 
 ## Development
 
-### Clone this Repository
+### Fork this Repository
+Follow Github's instructions on [how to fork a repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) to create your own fork.
+
+### Clone Your Repository
 
 ```bash
-git@gitlab.com:interlay/interbtc-api.git
+git@github.com:<your_github_profile>/interbtc-api.git
 cd interbtc-api
 ```
 
 ### Setting up a Local Development Environment
 
-You can spin up the parachain including the different clients with docker-compose:
+Start by setting your Node version via `nvm`
 
 ```bash
-docker-compose up
+nvm use
 ```
+(If necessary, `nvm` will guide you on how to install the version.)
 
-If you want to run components individually, you can clone the repositories and run the commands as done in `docker-compose.yml`.
-
-To install dependencies, run
+Next, install the dependencies
 
 ```bash
 yarn install
 ```
 
-Build the library using
+Finally, build the library
 
 ```bash
 yarn build
 ```
 
 ### Testing
 
-To run only unit tests, use
+To run unit tests only, use
 
 ```bash
 yarn test:unit
 ```
 
-Note that the parachain needs to be running for all tests to pass.
+#### Start the Parachain Locally for Integration/All Tests
+
+Note that the parachain needs to be running for all tests to run.
+
+```bash
+yarn docker-parachain-start
+```
 
-Then, to run tests, run
+The default parachain started is Kintsugi (KINT). To choose a specific chain, use the optional `--chain` parameter.
+e.g. to start Kintsugi explicitly
+
+```bash
+yarn docker-parachain-start --chain=KINT
+```
+
+Or, to start Interlay locally use
+
+```bash
+yarn docker-parachain-start --chain=INTR
+```
+
+When in doubt, start with Kintsugi. You will know when to use Interlay
```

Comment:
This sentence seems incomplete to me, do you think it's needed?

Reply:
It's a weird concept to try and communicate without context. I think you're right - we're better off removing it.